content/pages/cloud.json

```
@@ -51,27 +51,27 @@
       "items": [
         {
           "_template": "feature",
-          "headline": "Collaborate With the Team",
-          "subline": "Onboard your team; no GitHub accounts required.  Tina Cloud brings user-management, auth, roles, and a GraphQL API for your content.",
+          "headline": "Collaborate as a Team",
+          "subline": "Tina Cloud brings authentication, user management, roles. Onboard your team in no time, no GitHub accounts for editors required.",
           "media": {
             "src": "/img/user-management.png"
           }
         },
         {
           "_template": "feature",
-          "headline": "Move Faster With Tina Cloud",
-          "subline": "Visual editing + Git storage allows you to ship content faster with the safety of version-control.",
+          "headline": "Ship Content Faster",
           "media": {
             "src": "/img/visual-editing.gif"
-          }
+          },
+          "subline": "Visual editing greatly improves the productivity of content teams and make their job much more enjoyable. You team can work with full confidence, powerful Git Version Control has your back."
         },
         {
           "_template": "feature",
-          "headline": "GraphQL API",
-          "subline": "The Tina Content API provides a GraphQL interface to your repositories files where content is stored in open formats like Markdown and JSON, allowing for easy content re-use and migrations.",
+          "headline": "Keep your Data in Check",
           "media": {
             "src": "/img/graphQL.png"
-          }
+          },
+          "subline": "Tina Cloud Content API provides a GraphQL interface to your repositories files to bring data integrity and relationships to your filesystem.\nStore your content in open formats like Markdown and JSON to ease content re-use and migrations."
```

```suggestion
          "subline": "Tina Cloud Content API provides a GraphQL interface to your repositories' files, bringing data integrity and relationships to your filesystem.\nStore your content in open formats like Markdown and JSON to ease content re-use and migrations."
```

> bringing data integrity and relationships to your filesystem.

"Filesystem" throws me off here. Might just be me, but maybe this is clearer?:

> bringing data integrity and relationships to your files' content.

(Github has a GraphQL Api for files, but our values comes from querying the content within them).